### DISCUSSION TITLE
Access speed significantly slowed down

### DISCUSSION THREAD

**Discussion by zhaoyingquan** [minio/minio#18829] [Q&A]
My cluster has 6 nodes. One of the nodes was down. After two days, I found out and restarted the node, but the speed of accessing pictures was much slower. After an hour, the speed of accessing pictures became normal. Why is this?

**Comment by klauspost** [comment]
There is no real info to go on here, and I'm not going to go into guessing.

We offer performance analysis as part of our [subscriptions](https://min.io/pricing).
